.github/workflows/push-pr-release.yaml

```diff
@@ -34,6 +34,15 @@ jobs:
             grafana/xk6 build ${{ steps.version.outputs.k6 }} \
             --output "dist/sm-k6-${{ matrix.goos }}-${{ matrix.goarch }}" \
             --with github.com/grafana/xk6-sm=.
+      - name: Build with xk6 and gsm
+        run: |-
+          mkdir -p dist
+          docker run --rm -i -u "$(id -u):$(id -g)" -v "${PWD}:/xk6" \
+            -e "GOOS=${{ matrix.goos }}" -e "GOARCH=${{ matrix.goarch }}" \
+            grafana/xk6 build ${{ steps.version.outputs.k6 }} \
+            --output "dist/sm-k6-${{ matrix.goos }}-${{ matrix.goarch }}-gsm" \
+            --with github.com/grafana/xk6-sm=.
+            --with github.com/grafana/gsm-api-go-client
```

Comment:
Another nit, I think this will pull whatever is latest for `gsm-api-go-client` and thus not make the build reproducible, which is not awesome. I think hardcoding the version here is preferrable to let it run wild. I can open a follow-up PR teaching renovate to update the pin.

Reply:
Good point. The extension is still in a PR, so this wont even work. I'll work on getting a version for that

Reply:
Could we point to the commit hash in that case?